ci-operator/jobs/openshift/hypershift/openshift-hypershift-main-presubmits.yaml

```diff
@@ -757,6 +757,85 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-kubevirt-metal-conformance,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
should we always run this job? maybe it's better to only run if we explicitly trigger it

Reply:
Hmm. No, this was definitely not my intention. What did I miss? 🤔